modules/nw-multi-network-policy-ipv6-suppport.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-multi-network-policy-ipv6-support_{context}"]
+= Supporting multi-network policies in IPv6 networks
+
+The ICMPv6 Neighbor Discovery Protocol (NDP) plays a crucial role in IPv6 networks, facilitating the interaction between devices on the same link. NDP is a set of messages and processes that enable devices to discover and maintain information about neighboring nodes.
+
+The Cluster Network Operator (CNO) deploys the iptables implementation of multi-network policy when `useMultiNetworkPolicy` is set to `true`.
+
+To support multi-network policies in IPv6 networks the Cluster Network Operator deploys the set of rules defined in the following YAML.
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: multi-networkpolicy-custom-v6-rules
+  labels:
+    tier: node
+    app: multi-networkpolicy
+data:
+  custom-v6-rules.txt: |
+    # accept NDP
+    -p icmpv6 --icmpv6-type neighbor-solicitation -j ACCEPT <1>
+    -p icmpv6 --icmpv6-type neighbor-advertisement -j ACCEPT <2>
+    # accept RA/RS
+    -p icmpv6 --icmpv6-type router-solicitation -j ACCEPT <3>
+    -p icmpv6 --icmpv6-type router-advertisement -j ACCEPT <4>
+    # accept redirect
```

Comment:
At the moment, refirects  and packet-too-big packet types are not supported. 
https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/multus-networkpolicy/custom-iptables-rules.yaml

- https://github.com/openshift/cluster-network-operator/pull/2010